src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.cs

```diff
@@ -4,8 +4,9 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using Microsoft.CodeAnalysis.Completion;
-using Microsoft.CodeAnalysis.Completion.Providers;
+using Microsoft.CodeAnalysis.Completion.Triggers;
```

Comment:
Yeah, it felt weird having these in the main namespace because they're really seldom used. Extenders and consumers almost never need them. However, if you're building something like the Controller, you will.